src/test/java/com/epam/tc/hw2/ex1/HomePageTest.java

```diff
@@ -0,0 +1,75 @@
+        //1. open test site url
+        String url = "https://jdi-testing.github.io/jdi-light/index.html";
+        webDriver.navigate().to(url);
+        //2. assert browser title
+        softAssertions.assertThat(webDriver.getCurrentUrl()).isEqualTo(url);
+        softAssertions.assertThat(webDriver.getTitle()).isEqualTo("Home Page");
+        //3. perform login
+        webDriver.findElement(By.cssSelector("a[href='#']")).click();
+        webDriver.findElement(By.id("name")).sendKeys("Roman");
+        webDriver.findElement(By.id("password")).sendKeys("Jdi1234");
+        webDriver.findElement(By.id("login-button")).click();
+        //4. assert username is logged
+        WebElement webElement = webDriver.findElement(By.id("user-name"));
+        softAssertions.assertThat(webElement.getText()).isEqualTo("ROMAN IOVLEV");
+        //5. assert that there are 4 items on the header section are displayed and they have proper texts
+        List<WebElement> elementsOnHeaderSection = webDriver.findElements(By.cssSelector(".m-l8 > li > a"));
+        softAssertions.assertThat(elementsOnHeaderSection).hasSize(4);
+        List<String> actualTextListElements = elementsOnHeaderSection.stream().map(WebElement::getText)
+            .collect(Collectors.toList());
+        List<String> expectedTextListElements = Arrays.asList("HOME", "CONTACT FORM", "SERVICE", "METALS & COLORS");
+        softAssertions.assertThat(actualTextListElements).isEqualTo(expectedTextListElements);
+        //6. assert that there are 4 images on the Index Page and they are displayed
+        List<WebElement> imagesOnPage = webDriver.findElements(By.cssSelector(".benefit-icon"));
+        softAssertions.assertThat(imagesOnPage).hasSize(4);
+        //7. assert that there are 4 texts on the Index Page under icons and they have proper text
+        List<WebElement> textElementsUnderImages = webDriver.findElements(By.cssSelector(".benefit-txt"));
+        List<String> textListElementsUnderImages = textElementsUnderImages.stream().map(WebElement::getText)
+                .collect(Collectors.toList());
+        List<String> expectedTextUnderImages =
+                Stream.of("To include good practices\nand ideas from successful\nEPAM project",
+                "To be flexible and\ncustomizable", "To be multiplatform",
+                "Already have good base\n(about 20 internal and\nsome external projects),\nwish to get more…")
+                .collect(Collectors.toList());
+        softAssertions.assertThat(textListElementsUnderImages).isEqualTo(expectedTextUnderImages);
+        final String windowHandler = webDriver.getWindowHandle();
+        //8. assert that there is the iframe with “Frame Button” exist
+        webElement = webDriver.findElement(By.id("frame"));
+        softAssertions.assertThat(webElement.isDisplayed()).isTrue();
+        //9. switch to the iframe and check that there is “Frame Button” in the iframe
+        webDriver.switchTo().frame(0);
```

Comment:
Please use `swithTo().frame(WebElement)` method instead of one having index in arguments

Reply:
done